### PR TITLE
fix(ui-core): playable-overlay

### PR DESCRIPTION
**Pull request by yteslenk**

### CODE REVIEW

File: packages/wix-ui-core/src/components/video/players/Playable.tsx [hunk 1]
```diff
@@ -135,13 +136,13 @@ class PlayablePlayer extends React.PureComponent<
...
       src,
       playing,
+      poster,
       muted,
       title,
       showTitle,
       loop,
       volume,
       controls,
-      preload,
```
Comment (kupriyanenko) [inline packages/wix-ui-core/src/components/video/players/Playable.tsx]: I don't think that we need to remove the ability to pass `preload` explicitly.

File: packages/wix-ui-core/src/components/video/players/Playable.tsx [hunk 2]
```diff
@@ -209,6 +209,10 @@ class PlayablePlayer extends React.PureComponent<
 
     this.player.on(ENGINE_STATES.ENDED, () => {
       this.setState({ hasBeenPlayed: false });
+      if (controls) {
+        this._hidePlayableUI();
+      }
+      this.player.seekTo(0);
```
Comment (kupriyanenko) [inline packages/wix-ui-core/src/components/video/players/Playable.tsx]: why do we need it?

File: packages/wix-ui-core/src/components/video/players/Playable.tsx [hunk 3]
```diff
@@ -240,9 +260,13 @@ class PlayablePlayer extends React.PureComponent<
...
         />
-        {!this.state.hasBeenPlayed && poster && (
+        {!this.state.hasBeenPlayed && (
           <div
-            className={styles.cover}
+            className={
+              !poster
+                ? classNames(styles.cover, styles.transparentOverlay)
+                : styles.cover
+            }
```
Comment (kupriyanenko) [inline packages/wix-ui-core/src/components/video/players/Playable.tsx]:
```
className={classNames(styles.cover, {styles.transparentOverlay: !poster})}
```

File: packages/wix-ui-core/src/components/video/Video.st.css [hunk 4]
```diff
@@ -26,6 +26,10 @@
     cursor: pointer;
 }
 
+.cover.transparentOverlay {
+    background: transparent 50% 50% no-repeat;
```
Comment (kupriyanenko) [inline packages/wix-ui-core/src/components/video/Video.st.css]: why not just `background: transparent;`?

File: packages/wix-ui-core/src/components/video/players/Playable.tsx (hunk 3 above)
Comment (yteslenk) [reply packages/wix-ui-core/src/components/video/players/Playable.tsx]: useful knowledge)

Comment (kupriyanenko) [reply packages/wix-ui-core/src/components/video/players/Playable.tsx]: check more here: https://github.com/JedWatson/classnames

File: packages/wix-ui-core/src/components/video/players/Playable.tsx (hunk 1 above)
Comment (yteslenk) [reply packages/wix-ui-core/src/components/video/players/Playable.tsx]: I deleted from this position pleload, cause I need to reinitialized it below
const preload = !poster ? 'metadata' : this.props.preload;

Comment (yteslenk) [reply packages/wix-ui-core/src/components/video/players/Playable.tsx]: We don't lose an ability to pass it explicitly.

Comment (kupriyanenko) [reply packages/wix-ui-core/src/components/video/players/Playable.tsx]: you can override this value if poster is not provided, but respect it in another case.

Comment (kupriyanenko) [reply packages/wix-ui-core/src/components/video/players/Playable.tsx]: > We don't lose an ability to pass it explicitly.

from your code looks like we loose

File: packages/wix-ui-core/src/components/video/players/Playable.tsx (hunk 2 above)
Comment (kupriyanenko) [reply packages/wix-ui-core/src/components/video/players/Playable.tsx]: so, if we really need it, please add a comment in the code, and think, maybe we need to fix something on the playable side.

Comment (yteslenk) [reply packages/wix-ui-core/src/components/video/players/Playable.tsx]: SeekTo I use to get a first frame of video.
Conditional check I could remove. If user removes controls in editor stage, there is nothing to remove.

File: packages/wix-ui-core/src/components/video/Video.st.css (hunk 4 above)
Comment (yteslenk) [reply packages/wix-ui-core/src/components/video/Video.st.css]: Agree.

File: packages/wix-ui-core/src/components/video/players/Playable.tsx (hunk 2 above)
Comment (kupriyanenko) [reply packages/wix-ui-core/src/components/video/players/Playable.tsx]: > SeekTo I use to get a first frame of video.

for me, it looks like a hack, that you need to document in your code. And also looks like something that we need to fix on the playable side. please discuss with @BrooklynKing

Comment (kupriyanenko) [reply packages/wix-ui-core/src/components/video/players/Playable.tsx]: My bad, nothing to do here

**Comment by kupriyanenko** [comment]
@yteslenk I think in this PR we need to update playable version (package.json) as well.

**Pull request merged**